frontend/public/module/k8s/index.ts

```diff
@@ -35,10 +35,12 @@ export type ObjectReference = {
 export type ObjectMetadata = {
   name?: string;
   generateName?: string;
+  uid?: string;
   annotations?: {[key: string]: string},
   namespace?: string,
   labels?: {[key: string]: string},
   ownerReferences?: OwnerReference[],
+  deletionTimestamp?: string;
```

Comment:
I reverted this back to just a string. The type generation will be configurable to map this property to string.

Reply:
instead of a `Date`